### PR TITLE
Add link to JavaScript ESLint config

### PR DESCRIPTION
**Pull request by msteward**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Guides is a public repo for storing, among other things, public configuration fi
 ### Style guide configurations
 
 [Ruby](/style/ruby/.rubocop.yml)
+[JavaScript](https://github.com/BetterOfficeApps/eslint-config-lendesk)
```
Comment (mrleeio) [inline README.md:8]: We should be storing these in a .eslintrc file

I know you are following up with @RafeHatfield about this 👍

**Comment by matrinox** [comment]
@msteward I brought up merging the other 2 style guides into this one. Thoughts?

**Comment by msteward** [comment]
@matrinox Do you mean this one? https://github.com/BetterOfficeApps/front-end-styleguide/

**Comment by matrinox** [comment]
@msteward Yes and https://github.com/BetterOfficeApps/rubocop-definitions

File: README.md (hunk 1 above)
Comment (mrleeio) [reply README.md:8]: Just adding this here for future selves:

Rafe has approved moving all linter configs into a public repo.

Carry on.

**Pull request closed without merging**